ospo-mindmap/Content/ospomindmap.md

```diff
@@ -93,11 +166,18 @@ development team of the Linux Project
   - How to file bug reports
   - How to contribute basic code
 
-#### Optimize open outbound source contributions
+#### Create Internal Open Source Ambassadors
 
-### OS Community Engagement
+- Drive event sponsorship
+- Train speakers for coding forums
 
-#### Incubate and launch OS projects into the public sphere
+#### Contributes to Third-Party OSS Projects
+
+- Ensure maintainability of OSS projects
+- Develop new features upstream
+- Fix bugs upstream
```

Comment:
We can also add non code contributions, like:
- Adds documentation
- Reproduce issues
- Offer project management

Reply:
Added the following points:

- Improve documentation
- Triage or reproduce issues
- Offer project management